<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="rysujFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string> Narzedzia</string>
    </property>
    <widget class="QPushButton" name="cleanButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Czyść</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyjście</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>86</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> Rysowanie dynamiczne</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="kolorButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ustaw Kolor</string>
     </property>
    </widget>
    <widget class="QLabel" name="gruboscLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Grubość pędzla:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="gruboscSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>148</y>
       <width>35</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[Wersja rozszerzona programu]</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Tryb rysowania</string>
    </property>
    <widget class="QRadioButton" name="odcinekButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Odcinek</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="okragButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Okrag</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="elipsaButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Wielokąt na elipsie</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="krzywaBezieraButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Krzywa Beziera</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="IloscWierzcholkowSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QLabel" name="IloscWierzcholkow">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ilość wierzchołków:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="krzywaBsklejanaButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Krzywa B-sklejana</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="floodFillButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>112</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>FloodFill</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="scanLineButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>ScanLine - wielokaty</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="olowekButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ołówek (Własne)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Krzywe - tryb działania</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="rysowanieButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Rysowanie</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="przemieszczenieButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Przemieszczenie punktu</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="usuniecieButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Usunięcie punktu</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
